readme.md

```diff
@@ -1092,6 +1092,14 @@ echo $faker->nationalId; // 'V11223344'
 echo $faker->taxpayerIdentificationNumber; // 'J1234567891'
 ```
 
+### `Faker\Provider\fr_CH\Person`
```

Comment:
Shouldn't this be available in `it_CH` and `de_CH` as well?

Reply:
Absolutely, I completely forgot about that. How would you suggest I implement that ? I would like to avoid duplicating the code as much as possible. Should I move all the AVS13 number generation out of the localized provider ?

Reply:
We're following the PSR-4 standard to auto-load classes. They're static methods, so I'd say you can just call them?